specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/HttpRouteConfig_CreateOrUpdate.json

```
@@ -0,0 +1,130 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "examplerg",
+    "environmentName": "testcontainerenv",
+    "httpRouteName": "httproutefriendlyname",
+    "api-version": "2024-10-02-preview",
+    "httpRouteConfigEnvelope": {
+      "location": "East US",
+      "properties": {
+        "customDomains": [
+          {
+            "name": "example.com",
+            "bindingType": "SniEnabled",
+            "certificateId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/testcontainerenv/certificates/certificate-1"
+          }
+        ],
+        "rules": [
+          {
+            "targets": [
+              {
+                "containerAppName": "capp-1",
```

```suggestion
                "containerApp": "capp-1",
```
What did you decide to do with revisionName?

---

@Tratcher i like your suggestion on dropping the suffix, its cleaner!
I also would like to keep it consistent with current `ingress.traffic.revisionName`. Lets discuss this more tomorrow, leaving this discussion open for now.

---

this is done now
we will have `targets.containerApp`, `targets.revision` and `targets.label`